sdk/storage/Azure.Storage.Blobs.DataMovement/tests/BlobDataMovementTests.cs

```diff
@@ -11,14 +11,14 @@
 namespace Azure.Storage.Tests
 {
     /// <summary>
-    /// Base class for Blob tests
+    /// Base class for BlobDataMovement tests
     /// </summary>
     [ClientTestFixture(
     BlobClientOptions.ServiceVersion.V2020_06_12,
     BlobClientOptions.ServiceVersion.V2020_08_04,
     RecordingServiceVersion = BlobClientOptions.ServiceVersion.V2020_08_04,
     LiveServiceVersions = new object[] { BlobClientOptions.ServiceVersion.V2020_06_12 })]
-    public class BlobDataMovementTests : DataMovementTestBase
+    public class BlobDataMovementTests : BlobTestBase
```

Comment:
We shouldn't be doing this. If you need piece of functionality from BlobTestBase that means it should be somehow refactored into Commons.

Reply:
So should I just move the functions I used from BlobTestBase to Common and then put this back to using DataMovementTestBase?